db/app.js

```diff
@@ -10,10 +14,12 @@ app.get("/api", (req, res) => {
 
 app.get("/api/topics", getTopics);
 
-app.get("/api/articles/:article_id", getArticle);
+app.get("/api/articles/:article_id", getArticles);
```

Comment:
Same here - your endpoint is to get one article but your controller suggests you're getting multiple articles